src/Order.js

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+const Order = ({ orderType }) => {
+  const [Orders, setOrders] = useState(0);
+  const orderOne = () => {
+    setOrders(Orders + 1);
+  };
+  return (
+    <li>
+      {orderType}: {Orders} <RestaurantButton handleClick={orderOne} />
+    </li>
+  );
+};
+
+export default Order;
```

Comment:
you don't capitalise a variable name (first letter), what if a customer wants to remove an order he needs another button for moving